.github/workflows/docker-image.yml

```diff
@@ -7,6 +7,14 @@ on:
         description: 'Push to Docker Hub'
         required: true
         default: 'false'
+      tags:
+        description: 'Custom tags to use for the push'
+        required: false
+        default: ''
+      ref:
+        description: 'Custom ref to check out'
+        required: false
+        default: ${{ github.sha || github.ref }}
```

Comment:
This should mimic the defaults set in https://github.com/actions/checkout/blob/main/src/input-helper.ts#L64.